plugins/@grouparoo/mailchimp/__tests__/export/export-profile.ts

```diff
@@ -428,9 +464,27 @@ describe("mailchimp/exportProfile", () => {
         expect.objectContaining({ name: listThree.toLowerCase() }),
       ])
     );
+  });
 
-    const oldUser = await getUser(emails["alternativeEmail"]);
-    expect(oldUser).toBe(null);
+  test("can not delete a user if sync mode does not allow it but should remove the tags", async () => {
```

Comment:
For this test, we should assert that we are only removing the tags we know about (if possible).  eg: give the profile some other random tag, and ensure that it still exists after the sync